data/json/monsters.json

```diff
@@ -5040,6 +5040,36 @@
     "description":"A gargantuan protoplasmic blob, constantly reshaping, forming new pseudopods seemingly at will.  All over its body are eyes that form and disappear.  It looks at you with malice.",
     "flags":["SEES", "SMELLS", "SWIMS", "PLASTIC", "SLUDGEPROOF", "ACID_BLOOD", "ACID_PROOF", "NOHEAD", "REGENERATES_50", "ABSORBS", "REGEN_MORALE", "NOGIB"]
     },{
+    "type":"MONSTER",
+    "id":"mon_albino_penguin",
+    "name": "albino penguin",
+    "species":"NETHER",
+    "default_faction":"nether",
+    "symbol":"V",
+    "color":"white",
+    "size":"HUGE",
```

Comment:
Size large would be more fitting. At 8 feet tall, they should have mass more comparable to that of a cow than of a tank.